### PR TITLE
Adding Get Started Pages

### PR DESCRIPTION
**Pull request by ryankeairns**
This adds a set of pages that can be used as templates for adding real content across a variety or languages and topics.

The Rails page used here have dummy content in them, but they will be swapped over to real content before linking to this pages in the navigation.

The thought is to move quickly, push these pages out as-is, flesh out new content, then expose a link to the new `/get-started` and `/guides` pages via the core navigation.

Under 'Get Started' we'll have pages that collect content from a variety of sources into one view (link to new content, demos, tutorials, blog posts, etc.). Additionally, some pages will link to a user guide which is essentially an end-to-end reference on getting an app up and running with Habitat. They will be arranged under the root `/guides` directory and be visually presented/categorized (on the Guides home page) by Language, Topic, Databases, Web Servers, etc.

The primary content we drive new users towards will be Get Started `/get-started`, but it will link back to Guides and vice-versa. Think of Get Started pages as broader in scope with less depth, while Guides are narrowly focused and deep - more like reference documentation or detailed tutorial.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/311865335).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/311865335).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by ryankeairns** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/312092529).

File: www/source/guides/rails/index.html.slim [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
```
[the remark this answers is not in the document]
Comment (ryankeairns) [reply www/source/guides/rails/index.html.slim]: We could do it either way, what I have at this point are really just suggestions on how to build the pages out. Our docs are also a mix of html and markdown. Point being, we can merge this PR and not be committed any one way... these Rails pages could simply be replaced with whichever format the author (or team) prefers.

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/312092529).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by cnunciato** [comment]
tenor-205613194

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/312163996).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/312163996).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**